node/packages/parser/src/ethereum/index.ts

```diff
@@ -19,12 +19,15 @@ export interface TransactionParserArgs {
   rpcUrl: string
 }
 
+export type Parser = thor.Parser | uniV2.Parser | zrx.Parser
+
 export class TransactionParser {
   network: Network
 
-  private thor: thor.Parser
-  private uniV2: uniV2.Parser
-  private zrx: zrx.Parser
+  private readonly thor: thor.Parser
+  private readonly uniV2: uniV2.Parser
+  private readonly zrx: zrx.Parser
```

Comment:
We probably don't need to keep track of these individually anymore.

Reply:
We do still use `this.thor` in `getInternalAddress`, and it felt a little weird keeping that property on the `TransactionParser` class, but removing the rest. Thoughts?

Reply:
Oh yeah, you are right. This is currently being used in the ingester which will be ripped out at some point, but until then let's just leave it as is.